### PR TITLE
[F] LRAC-14342 update unit tests

### PR DESCRIPTION
**Pull request by fmarcio**
Jira issue https://liferay.atlassian.net/browse/LRAC-14342

### CODE REVIEW

**Comment by fmarcio** [comment]
ci:test:sf

**Comment by ViniLopes20** [comment]
ci:test:relevant

**Comment by ViniLopes20** [comment]
Hey @fmarcio, the failure of `relevant` is not related to your modifications. The PR looks good to be forwarded. Thanks!

**Comment by fmarcio** [comment]
> Hey @fmarcio, the failure of `relevant` is not related to your modifications. The PR looks good to be forwarded. Thanks!

@ViniLopes20 I'll check with @interaminense if he approves this PR. After that I'll forward it. Thanks vini

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/experiments/hocs/mappers/__tests__/__snapshots__/experiment-summary-mapper.js.snap [hunk 1]
```diff
@@ -5,41 +5,22 @@ Object {
   "alert": Object {
     "description": "We recommend that you publish the winning variant.",
     "symbol": "check-circle",
-    "title": "Variant 02 has outperformed control by at least 5%",
+    "title": "Variant 01 has outperformed control by at least 1%",
```
[the remark this answers is not in the document]
Comment (fmarcio) [reply modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/experiments/hocs/mappers/__tests__/__snapshots__/experiment-summary-mapper.js.snap:8]: Ok @manuferreira I'll update it. Thx

**Comment by fmarcio** [comment]
ci:test:sf

**Comment by fmarcio** [comment]
ci:test:relevant

File: modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/experiments/hocs/mappers/__tests__/__snapshots__/experiment-summary-mapper.js.snap (hunk 1 above)
[the remark this answers is not in the document]
Comment (manuferreira) [reply modules/dxp/apps/osb/osb-faro/osb-faro-web/src/main/js/experiments/hocs/mappers/__tests__/__snapshots__/experiment-summary-mapper.js.snap:8]: Thank you! :muscle:

**Comment by manuferreira** [comment]
nvm, @fmarcio has to add the `by` because it's on the mockup, not sure how the `by` disappeared when I was testing locally. Thanks again :eyes:

**Comment by fmarcio** [comment]
ci:test:sf

**Comment by fmarcio** [comment]
ci:test:relevant

**Review by interaminense — APPROVED** [review]
LGTM!

**Comment by manuferreira** [comment]
Hey @fmarcio, I don't think the relevant errors are related to your PR, you good to go.

**Comment by fmarcio** [comment]
ci:forward:force

**Pull request closed without merging**